src/components/SidebarList.js

```diff
@@ -112,17 +149,29 @@ export default class SidebarList extends React.Component {
 
     return (
       <div className="spec sidebar-list" id="spec-sidebar-list">
-        <ul>
+        <ul role="list">
           <li className="spec list-title">Resources</li>
           <FilterContainer />
           {this.state.filteredSidebarData.map((sidebarItem, tag) =>
             <li className={"submenu " + this.ifActive(this.isTagActive(tag))} >
-              <span className="submenu-title" onClick={() => this.subMenuClicked(tag)}>{tag}</span>
-              <ul className="submenu-items">
+              <span
+                role="listitem" tabIndex={0} className="submenu-title"
+                onClick={() => this.subMenuClicked(tag)}
+                onKeyUp={(e) => this.subMenuKeyup(e.key, tag)}
+              >
+                {tag}
+              </span>
+              <ul className="submenu-items" role="menu">
                 {sidebarItem.get("operations").map(op =>
                   <div>
-                    <li className={"method " + this.ifActive(this.isIdActive(op.get("id")))} >
-                      <a onClick={() => this.sidebarAnchorClicked(tag, op)} className={"method-" + op.get("method")}>
+                    <li tabIndex={this.isTagActive(tag) ? 0 : -1}
```

Comment:
non-blocking nitpick/question:  should `tabIndex` be on a new line? There seems to be some variation in styling, not just in this file, but also in other places throughout Swagger-ui.

Reply:
I've definitely seen my share of both styles in different projects. I think starting attributes that need to wrap around on the next line does look a little cleaner. Happy to go with that as the way towards consistency.